src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements FigureBehaviour {

```

Comment:
You missed one behavior:
![image](https://github.com/mate-academy/jv-oop-advanced/assets/41166578/f62548fd-03f9-4903-93a6-f599525e4999)
```suggestion
public abstract class Figure implements AreaCalculator, Drawable {
```

```
public interface AreaCalculator {
    double getArea();
}
```